GoogleUtilities/ISASwizzler/GULObjectSwizzler.m

```diff
@@ -98,10 +104,9 @@ - (void)copySelector:(SEL)selector fromClass:(Class)aClass isClassSelector:(BOOL
                                   : class_getInstanceMethod(aClass, selector);
   Class targetClass = isClassSelector ? object_getClass(_generatedClass) : _generatedClass;
   IMP implementation = method_getImplementation(method);
+
   const char *typeEncoding = method_getTypeEncoding(method);
-  BOOL success __unused = class_addMethod(targetClass, selector, implementation, typeEncoding);
-  NSAssert(success, @"Unable to add selector %@ to class %@", NSStringFromSelector(selector),
-           NSStringFromClass(targetClass));
+  class_replaceMethod(targetClass, selector, implementation, typeEncoding);
```

Comment:
Are there any cleanups to the older method that needs to be done before replacing to the new implementation?

With addMethod, we would fail if an implementation already existed. With this change, we might mask the failure and force replace with the new implementation.

Reply:
My assumption here was that replacing the method implementation is a preferable action. 

What type of handling we would prefer to have here? We can reintroduce the debug assertion in the case the method was replaced or we can return previous behaviour - silently don't replace the implementation and assert under DEBUG only.